### PR TITLE
Allow you to choose your pronouns at will.

### PR DESCRIPTION
**Pull request by ShiftyRail**
It is, after all, 2018.

:cl:
- tweak: You can now choose which pronouns should be displayed on examine with the verb "Choose pronouns" in the IC tab.

### CODE REVIEW

**Comment by nuklearcellphoneg** [comment]
Only if it gives 9999 brain damage for changing the displayed gender :^)

**Comment by ghost** [comment]
I don't like this. The 256 character limit is too oppressive. After all the game is set in 2710.

**Comment by ghost** [comment]
Time for traps :DDDDDD

**Comment by ShiftyRail** [comment]
Whomever downvoted this isn't very progressive-minded

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -1,6 +1,29 @@
...
+
+/mob/living/carbon/human/verb/choose_pronouns()
+	set name = "Choose pronouns"
+	set category = "IC"
+
+	pronoun_personal_subject = stripped_input(src, "Please enter your desired personal pronoun.", "Personal pronoun", "They", 5)
+	pronoun_possesive = stripped_input(src, "Please enter your desired possesive pronoun.", "Possesive pronoun", "Their", 5)
+	pronoun_reflexive = stripped_input(src, "Please enter your desired reflexive pronoun.", "Reflexive pronoun", "Them", 5)
+	verb_be = alert(src, "How does your desired pronoun conjugates with the auxiliary 'be'?", "Auxiliary 'to be'", "is", "are")
+	verb_have = alert(src, "How does your desired pronoun conjugates with the auxiliary 'be'?", "Auxiliary 'to have'", "has", "have")
```
Comment (Intigracy) [inline code/modules/mob/living/carbon/human/examine.dm]: conjugate, not conjugates
and the second one should be 'have'

Comment (ShiftyRail) [reply code/modules/mob/living/carbon/human/examine.dm]: Thanks !

**Comment by FudgePucker** [comment]
Great April fools PR.

Should gib you the rest of the year.

**Pull request merged**